UnitConverter/UnitConverter.swift

```diff
@@ -9,8 +9,8 @@ import Foundation
 
 class UnitConverter {
     
-    func degreesFahrenheit(degreesCelsius: Int) -> Int {
-        return Int(1.8 * Float(degreesCelsius) + 32.0)
+    func degreesFahrenheit(degreesCelsius: Int) -> Float {
```

Comment:
I agree with the intent here, but for the sake of this app/lesson, I intentionally kept it to `Int`. My concern is that, by making this a `Float`, then we also need to handle the string formatting to keep it to one decimal place in the view. By keeping it an `Int`, we don't have to care. (Students already learn about string formatting a decimal in [Stopwatch](https://github.com/SwiftEducation/Stopwatch)).